### DISCUSSION TITLE
Liberation 5 - Marianas Map No Hornets?

### DISCUSSION THREAD

**Discussion by GrEaSeLiTeNiN** [Khopa/dcs_liberation#1737] [Q&A] [answered]
The first two Marianas campaigns do not have F-18s in any of the squadrons list even though the initial set up process shows they are in the list of available aircraft. Only the pure air (3rd) campaign one has them. Did I miss something? Thanks

**Comment by Starfire13** [comment, marked as answer]
Squadrons are now preset by the campaign designer, so only specific aircraft are available. If you wish to change the defaults, it is a simple matter of editing the campaign YAML file in any text editor and changing the name of the aircraft type. Check out the wiki for info on how to do this.

**Reply by GrEaSeLiTeNiN** [reply]
So shouldn't the list of available aircraft be corrected to exclude the Hornets if they are going to be unavailable?

**Reply by Starfire13** [reply]
No, because those are the aircraft that are available to that faction. As an example, the US Air Force has lotsa different aircraft. But you don't see every single aircraft in the US Air Force's inventory at every air base. The campaign designer can pick the aircraft he wants from that list, and set squadrons at the various airfields/carriers in the campaign. However, you don't *have* to follow those defaults, and can edit the YAML file and pick any other aircraft available to that faction that you would rather use for that squadron.

**Reply by GrEaSeLiTeNiN** [reply]
Ok thanks, I get it... what "available" actually means in this context.
